src/WebJobs.Script/Description/DotNet/DotNetFunctionInvoker.cs

```diff
@@ -170,21 +170,32 @@ private void ResetFunctionValue()
 
         private void RestorePackages()
         {
+            // Kick off the package restore and return.
+            // Any errors will be logged in RestorePackagesAsync
+            RestorePackagesAsync(true)
```

Comment:
We haven't made the changes to restore from the primary instance only yet, so I didn't want to introduce that with this PR. Would probably be a good idea to have that done in a separate set of changes. Thoughts?